src/components/Stream/Stream.js

```diff
@@ -1,9 +1,67 @@
 import React from 'react';
+import videos from '../../../public/TempVideos/Videos';
 
 const Stream = () => {
+
+    const handleMouseEnter = (event) => {
+        const video = event.target;
+        if (document.contains(video)) {
+            video.play().catch(error => {
+                console.error('Error attempting to play video:', error);
+            });
+            video.style.transform = 'scale(1.1)';
+            video.style.zIndex = 1;
+        }
+    };
+
+    const handleMouseLeave = (event) => {
+        const video = event.target;
+        if (document.contains(video)) {
+            video.pause();
+            video.style.transform = 'scale(1)';
+            video.style.zIndex = 0;
+        }
+    };
+
+    const handleClick = (event) => {
+        const video = event.target;
+        if (document.contains(video)) {
+            if (video.requestFullscreen) {
+                video.requestFullscreen();
+            } else if (video.mozRequestFullScreen) {
+                video.mozRequestFullScreen();
+            } else if (video.webkitRequestFullscreen) { 
+                video.webkitRequestFullscreen();
+            } else if (video.msRequestFullscreen) {
+                video.msRequestFullscreen();
+            }
+        }
+    };
```

Comment:
Nitpick: this can be simplified
```suggestion
    const handleClick = ({ target: video }) => {
    if (document.contains(video)) {
        const requestFullscreen = 
            video.requestFullscreen || 
            video.mozRequestFullScreen || 
            video.webkitRequestFullscreen || 
            video.msRequestFullscreen;

        requestFullscreen?.call(video);
    }
};
```

Reply:
Agreed, apologies. Committed the recommended changes

Reply:
Looks good now 👑